setup.conf

```diff
@@ -1,28 +1,35 @@
-<SETUP_CONF.ROLE_ARN.EXTERNAL_SECRETS_SERVICE>=arn:aws:iam::863518836478:role/dev-kf13_external-secrets
-<SETUP_CONF.ROLE_ARN.EXTERNAL_SECRETS.AWS>=arn:aws:iam::863518836478:role/dev-kf13_kubeflow_external-secrets-kubeflow
-<SETUP_CONF.CLUSTER_NAME>=dev-kf13
-<SETUP_CONF.AWS_REGION>=eu-central-1
-<SETUP_CONF.VPC_ID>=vpc-038b1e8ba71bdbecf
-<SETUP_CONF.DOMAIN>=dev-kf13.learn-mlops.com
-<SETUP_CONF.SUBDOMAIN_DASHBOARD>=dashboard
-<SETUP_CONF.SUBDOMAIN_SERVING>=serving
-<SETUP_CONF.SUBDOMAIN_ARGOCD>=gitops
-<SETUP_CONF.RDS.DB_NAME.CACHE>=cachedb
-<SETUP_CONF.RDS.DB_NAME.PIPELINES>=mlpipeline
-<SETUP_CONF.RDS.DB_NAME.METADATA>=metadb
-<SETUP_CONF.RDS.DB_NAME.KATIB>=katib
-<SETUP_CONF.RDS.DB_NAME.MLFLOW>=mlflow
-<SETUP_CONF.RDS.HOST>=dev-kf13-instance.c6e2xeayekua.eu-central-1.rds.amazonaws.com
-<SETUP_CONF.RDS.PORT>=3306
-<SETUP_CONF.ARTIFACT_S3_BUCKET>=863518836478-dev-kf13-kubeflow
-<SETUP_CONF.ACM_CERTIFICATE_ARN>=arn:aws:acm:eu-central-1:863518836478:certificate/135b5536-9272-47a4-a438-3fe2f78801c0
-<SETUP_CONF.COGNITO_USER_POOL.ARN>=arn:aws:cognito-idp:eu-central-1:863518836478:userpool/eu-central-1_fK540dhFK
-<SETUP_CONF.COGNITO_USER_POOL.CLIENT_ID>=3pvirkutha95pinlfpv1cldvgd
-<SETUP_CONF.COGNITO_USER_POOL.DOMAIN>=863518836478-dev-kf13
-<SETUP_CONF.EXTERNAL_SECRET_NAME.AWS.RDS_USERNAME>=dev-kf13/kubeflow/rds_username
-<SETUP_CONF.EXTERNAL_SECRET_NAME.AWS.RDS_PASSWORD>=dev-kf13/kubeflow/rds_password
-<SETUP_CONF.EXTERNAL_SECRET_NAME.AWS.S3_ACCESS_KEY>=dev-kf13/kubeflow/s3_access_key
-<SETUP_CONF.EXTERNAL_SECRET_NAME.AWS.S3_SECRET_KEY>=dev-kf13/kubeflow/s3_secret_key
+<<__role_arn.argocd__>>=arn:aws:iam::863518836478:role/dev-kf13-3_argocd_argocd
+<<__role_arn.cluster_autoscaler__>>=arn:aws:iam::863518836478:role/dev-kf13-3_kube-system_aws-cluster-autoscaler
+<<__role_arn.loadbalancer_controller__>>=arn:aws:iam::863518836478:role/dev-kf13-3_kube-system_aws-loadbalancer-controller
+<<__role_arn.external_dns__>>=arn:aws:iam::863518836478:role/dev-kf13-3_kube-system_external-dns
+<<__role_arn.external_secrets_service__>>=arn:aws:iam::863518836478:role/dev-kf13-3_kube-system_external_secrets
+<<__role_arn.external_secrets.mlflow__>>=arn:aws:iam::863518836478:role/dev-kf13-3_mlflow
+<<__role_arn.external_secrets.kubeflow__>>=arn:aws:iam::863518836478:role/dev-kf13-3_kubeflow
+<<__role_arn.external_secrets.git_repo__>>=arn:aws:iam::863518836478:role/dev-kf13-3_argocd
+<<__external_secret_name.mlflow.rds_username__>>=dev-kf13-3/mlflow/rds_username
+<<__external_secret_name.mlflow.rds_password__>>=dev-kf13-3/mlflow/rds_password
+<<__external_secret_name.kubeflow.rds_username__>>=dev-kf13-3/kubeflow/rds_username
+<<__external_secret_name.kubeflow.rds_password__>>=dev-kf13-3/kubeflow/rds_password
+<<__external_secret_name.git_repo.https_username__>>=dev-kf13-3/argocd/https_username
+<<__external_secret_name.git_repo.https_password__>>=dev-kf13-3/argocd/https_password
+<<__git_repo.url__>>=https://gitlab.alexanderthamm.com/kaas/at-argocd.git
```

Comment:
This will need to be changed to this main repo, as well as some other settings in this file that are specific to your current deployment.

Reply:
This is meant as an example while we develop. We should in fact probably move it to an "example" folder